docs/src/how_it_works.md

```diff
@@ -0,0 +1,77 @@
+# How it Works (By Example)
+
+## Example Problem
+
+Let $f$ be a function of position ($\bar{r}$)
+```julia
+function f(r̄::Meshes.Point)
+    x, y, z = to(r̄)
+    ...
+end
+```
+
+Let the integration domain be the space (a ball) enclosed by a sphere centered on the origin with a radius of 5 meters.
+```julia
+center = Meshes.Point(0u"m", 0u"m", 0u"m")
+radius = 5.0
+ball = Meshes.Ball(center, radius)
+```
+
+This integral is often expressed abstractly as simply the following, where the triple integral signs and $\text{d}V$ indicate that the integration domain is some three-dimensional volume.
+```math
+\iiint f(\bar{r}) ~ \text{d}V
+```
+
+Integrals like this are often solved manually by selecting an appropriate coordinate system and limits that neatly represent the integration domain, e.g.
+```math
+\int_0^{\pi} \int_0^{2\pi} \int_0^{5} f(\bar{r}) ~ \text{d}\rho~\text{d}\theta~\text{d}\phi
+```
+
+This works great for simple geometries, but requires integration code that is geometry-specific. This package, by contrast, leverages parametric functions defined in Meshes.jl and differential forms to define integral methods that are general solutions for all geometries.
+
+## Parametric Functions
+
+Every supported `Meshes.Geometry` type is defined as having a parametric function that maps from a local parametric coordinate system to every point on the geometry. Curve-like geometries will have a single parametric dimension, surfaces will have two dimensions, and volumes will have three dimensions; this can be checked for a particular geometry via `Meshes.paramdim(geometry)`.
+
+For consistency across geometry types, these parametric functions are defined to always take coordinates inside a normalized range $[0,1]$. In the example case of `ball`, Meshes.jl defines a parametric function mapped in normalized spherical coordinates $(t_\rho, ~t_\theta, ~t_\phi)$. We find, then:
```

Comment:
> these parametric functions are defined to always take coordinates inside a normalized range $[0,1]$

Well, except for `Line` and co. There some of the parametric coordinates can go to infinity in one or both directions.

Reply:
I just pushed a commit that addresses this. I changed the Triangle-specific page into a Specializations page with a header that outlines the geometries that don't conform to this "standard". I tweaked the language here to caveat "with some notable exceptions" and provide a link to Specializations. Hopefully the new `@ref` links work. (Not a polished solution, but hopefully good enough for today.)

Reply:
Let me know what you think of the update.

Reply:
Sounds good. I left one suggestion to avoid confusion.